contracts/mainnet/connectors/merkleDistributor/main.sol

```diff
@@ -0,0 +1,66 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+
+import { Basic } from "../../common/basic.sol";
+import { Events } from "./events.sol";
+import { IFluidMerkleDistributor } from "./interface.sol";
+
+abstract contract FluidMerkle is Basic, Events {
+
+    address private constant MERKLE_DISTRIBUTOR = address(0);
+    
+    /**
+     * @dev Claims rewards from Fluid merkle distributor contract
+     * @param cumulativeAmount_ Total reward cumulated since reward inception.
+     * @param fToken_ Address of fToken on which rewards are being distributed.
+     * @param cycle_ Current epoch cycle.
+     * @param merkleProof_ Merkle proof that validates this claim.
+     * @param getId_ Id to get the cumulative amount.
+     */
+    function claim(
+        uint256 cumulativeAmount_,
+        address fToken_,
+        uint256 cycle_,
+        bytes32[] calldata merkleProof_,
+        uint256 getId_
```

Comment:
@shriyatyagii We can completely remove getId i guess. Or we can add setId along with it? 

reward amount user is gonna get is not always eq to cumulativeAmount_.

Reply:
Oh got it! Makes sense to add setId then